### ISSUE TITLE
Weird thing in post reply and in quick reply

### ISSUE THREAD

**Issue by Cu8eeeR** [WildcardSearch/MentionMe#87]
Two issues (latest 1.8 release branch).

- quick reply - it shows a spinner (still loading, nothing happens)
- new reply - type box is shown at the bottom (check image attached)

![bug](https://cloud.githubusercontent.com/assets/778890/21072763/b0a2f7b8-becd-11e6-8a86-1b012898bd99.png)

**Comment by WildcardSearch** [comment]
Can you check to see if `mentionme.css` exists in your theme's stylsheets in ACP?

Try editing the style sheet and saving it and see if that fixes the problem.

**Comment by Cu8eeeR** [comment]
@WildcardSearch - OK, mentionme.css is added correctly. So I just resaved it, it works now better.

Anyway, the dropdown window will need some CSS love :) I think you know it. Everything else works fine (At least everything what I have tested so far).

![css](https://cloud.githubusercontent.com/assets/778890/21074415/c79ffdaa-bef8-11e6-9065-0f34affa46b6.png)

**Comment by Cu8eeeR** [comment]
Would be nice to have highlighted letters in usernames
http://i.imgur.com/WiE0XgP.gif

**Comment by WildcardSearch** [comment]
> I just resaved it, it works now better

This is a recurring thing. I need to figure out why that style sheet isn't getting cached.

> the dropdown window will need some CSS love :) I think you know it.

Trust me I know that am not a designer 👅 I'd welcome any help making things look better.

> Would be nice to have highlighted letters in usernames

You mean highlight the letters that match?

**Comment by Cu8eeeR** [comment]
> You mean highlight the letters that match?

Yes, I am sorry for my English, not a native speaker

> This is a recurring thing. I need to figure out why that style sheet isn't getting cached.

I will try again on a fresh local board

> Trust me I know that am not a designer 👅 I'd welcome any help making things look better.

I would like to help you but I think that my skill for CSS and JS are not great :( I am trying my best but a lot of still need to be learn

**Comment by WildcardSearch** [comment]
There is still a good bit that I want to do before release. One thing being to get some continuity between the autocomplete for SCEditor and the autocomplete for Quick Reply/normal text areas.

Once that is done, everything will be in templates so perhaps I can get some help adjusting the HTML/CSS so that we can release with a better looking popup 👍

**Comment by Cu8eeeR** [comment]
OK, tested again. Works on first install (I am not sure what was wrong before) - the CSS loading part.

I sent you a PM on mybb.com - just test whatever you need :)

**Comment by Cu8eeeR** [comment]
@WildcardSearch - I have some ideas:
* when you type @ - show the usernames from users who contributed in the thread first (xenforo has this feature, quite useful)
* full text search (when you type "er" it will show also user "Tester"

**Comment by WildcardSearch** [comment]
> OK, tested again. Works on first install (I am not sure what was wrong before) - the CSS loading part.

I don't know either. I'm going to have a look at the way I am installing the style sheet and make sure it matches to the core code.

> I sent you a PM on mybb.com - just test whatever you need :)

Great. Thanks.

> when you type @ - show the usernames from users who contributed in the thread first

I agree. In fact #55 is confirmed for 3.1 😄 

> full text search

Another great idea. I'll make an issue for it.

**Comment by WildcardSearch** [comment]
I'm going to close this discussion thread as release nears.
